jobs/build/build-sync/Jenkinsfile

```diff
@@ -35,7 +35,7 @@ node {
     // Location of the image stream to apply
     def ocIsObject = "${mirrorWorking}/release-is.yaml"
     // Locally stored image stream stub
-    def baseImageStream = "/home/jenkins/base-art-latest-imagestream.yaml"
+    def baseImageStream = "/home/jenkins/base-art-latest-imagestream-${params.BUILD_VERSION}.yaml"
```

Comment:
If you need a place to write to you should move this to `/mnt/workspace`. Create a dir in there if you want just remember to `sudo chown jenkins:jenkins` it as the base dir is root owned

Reply:
I want to do that kind of thing in a followup PR.

Reply:
(For reference, both the 4.0 and 4.1 files exist already in the path I modify in this PR)

Reply:
ack... sounds good